packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -32,35 +33,53 @@ import CardHeader from '../../../components/CardHeader';
 
 export const viewName = 'complete-reset-password';
 
-export type CompleteResetPasswordProps = {
-  email: string;
-  forceAuth?: boolean;
-  linkStatus: LinkStatus;
-  // resetPasswordConfirm will be obtained from the relier
-  // resetPasswordConfirm determines if sync warning is shown
-  resetPasswordConfirm?: boolean;
-};
-
 type FormData = {
   newPassword: string;
   confirmPassword: string;
 };
 
 type LinkStatus = 'expired' | 'damaged' | 'valid';
 
-const CompleteResetPassword = ({
-  email,
-  forceAuth = false,
-  linkStatus,
-  resetPasswordConfirm,
-}: CompleteResetPasswordProps & RouteComponentProps) => {
-  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+const CompleteResetPassword = (_: RouteComponentProps) => {
+  logPageViewEvent(viewName, REACT_ENTRYPOINT);
 
   const [passwordMatchErrorText, setPasswordMatchErrorText] =
     useState<string>('');
-  const alertBar = useAlertBar();
   const navigate = useNavigate();
-  const ftlMsgResolver = useFtlMsgResolver();
+  const account = useAccount();
+  const [linkStatus, setLinkStatus] = useState<LinkStatus>('valid');
+  const [errorCompletePwdReset, setErrorCompletePwdReset] =
+    useState<boolean>(false);
+
+  // TODO: Pull this information from relier, in meantime we can get from query params
```

Comment:
Is this not available with Dan's PR merged now?

Reply:
It is but not 100%, I'll still need to add a few things for password reset state. Easier to pull into a new issue.

Reply:
Cool, I'm thinking we might need an epic for tasks/bugs we need to work on before certain flows are pushed to prod that the follow up mentions can probably go in.